snowflake_notebooks/README.md

```diff
@@ -0,0 +1,48 @@
+# Intro to Machine Learning with Snowpark ML for Python
+## Using [Snowflake Notebooks!](https://docs.snowflake.com/en/LIMITEDACCESS/snowsight-notebooks/ui-snowsight-notebooks-about)
+
+Want to try out the Intro to Machine Learning with Snowpark ML Quickstart using Snowflake's Notebooks in Snowsight? This folder contains three notebooks that are ready-to-use in your Snowflake environment:
+- [1_sf_nb_snowpark_ml_data_ingest.ipynb](./1_sf_nb_snowpark_ml_data_ingest.ipynb)
+- [2_sf_nb_snowpark_ml_feature_transformations.ipynb](./2_sf_nb_snowpark_ml_feature_transformations.ipynb)
+- [3_sf_nb_snowpark_ml_model_training_inference.ipynb](./3_sf_nb_snowpark_ml_model_training_inference.ipynb)
+
+To get started using Snowflake Notebooks, first login to Snowsight and run the following [setup.sql](./setup.sql) in a SQL worksheet (we need  to create the database, warehouse, schema, etc. that we will use for our ML project).
```

Comment:
Why do you need to do this in a SQL worksheet instead of using SQL cells in a notebook? IMO this is one of the nice advantages of using notebooks

Reply:
Yeah the main issue here is that most of the SQL commands involve creating the DB, schema, and warehouse where the notebook is also going to run.. I'm making the assumption that a customer could be starting from a fresh/blank Snowflake account, so they have to create the DB/schema/warehouse at a minimum **somewhere**.

We could definitely move some of the commands to the notebook- e.g. creating the file format, external stage, etc., but a certain amount of the setup I think has to be in SQL worksheet still, no?

Reply:
Yes the minimal admin stuff has to be done here and probably uses a different role, it makes sense to keep that part separate. But anything related to the demo I'd add into the notebook

Reply:
If you move the data staging into the notebook they'll have to set up an EAI though :/ so maybe just leave it as is for now. In an ideal world we'd be able to show off some of the power of doing more SQL via the notebook itself but maybe thats impractical for now.

Reply:
yeah makes sense- in notebook 3, I included showing how to call models deployed in registry using SQL (and a SQL cell) to try and highlight this- in the original quickstart we have to use `session.sql`, but we highlight using a SQL cell specifically there

if there are other opportunities (in the first notebook in particular) to highlight SQL I'm open to modifying some to include that

Reply:
agree on avoiding EAI setup/etc. in this example, it overcomplicates things IMO